app/controllers/administrator_controller.rb

```diff
@@ -40,4 +42,42 @@ def changeBasic
     end
   end
 
+  def sort
```

Comment:
I think this method is not needed, all pagination/sorting is handled in JS by the table/pagination plugin. Please confirm and delete this method if so.